test/v2_test.dart

```diff
@@ -1,96 +1,115 @@
 // Copyright (c) 2017, joeconway. All rights reserved. Use of this source code
 // is governed by a BSD-style license that can be found in the LICENSE file.
 
-import 'package:open_api/v2.dart';
-import 'package:test/test.dart';
-import 'dart:io';
 import 'dart:convert';
+import 'dart:io';
+
+import 'package:conduit_open_api/v2.dart';
+import 'package:dcli/dcli.dart';
+import 'package:test/test.dart';
 
 void main() {
-  group("Kubernetes spec", () {
+  group("Petstore spec", () {
     APIDocument? doc;
     Map<String, dynamic>? original;
 
     setUpAll(() {
-      // Spec file is too large for pub, and no other way to remove from pub publish
-      // than putting in .gitignore. Therefore, this file must be downloaded locally
-      // to this path, from this path: https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/swagger.json.
-      var file = File("test/specs/kubernetes.json");
-      var contents = file.readAsStringSync();
-      original = json.decode(contents);
-      doc = APIDocument.fromMap(original!);
+      /// download sample api document if we don't already have it.
+      final String config = fetchPetStoreExample();
+      final file = File(config);
+      final contents = file.readAsStringSync();
+      original = json.decode(contents) as Map<String, dynamic>;
+      if (original != null) {
```

Comment:
Not sure when this ends up null. If it does, then someone set up the tests wrong.

Reply:
I will remove the test.